vaapi/vaapisurfaceallocator.cpp

```diff
@@ -26,8 +26,8 @@
 namespace YamiMediaCodec{
 
 VaapiSurfaceAllocator::VaapiSurfaceAllocator(VADisplay display, uint32_t extraSize)
-    :m_display(display),
-    m_extraSize(extraSize)
+    : m_display(display)
```

Comment:
why we need change this file?

Reply:
it is changed by clang, because I add one NULL line.